GillianCore/GIL_Syntax/Annot_old.ml

```diff
@@ -0,0 +1,41 @@
+(** {b GIL annot}. *)
```

Comment:
Why are we keeping this around?

Reply:
Oops, I kept it around for reference and forgot to delete before committing :P
Removed in 28844c5eb93bc239e8ee95b2845bd02edb5aadbe